physiology-texts-vs-real-world.md

```diff
@@ -3,7 +3,8 @@
 ---
-A sentence taken out of context is likely to be misleading. A chemical equation that is concerned only with the reactants, catalyst, and product, can be misleading, and its industrial application is likely to produce devastation and pollution along with the intended product. In nature and industry, the reactants, products, and energy changes are linked to the ecology and to the economy. In physiological chemistry, events in the organism are linked to the environment so closely that food, water, air, soil, and pollution form a firmly linked functional system.
+
+changes are comming from an seo A sentence taken out of context is likely to be misleading. A chemical equation that is concerned only with the reactants, catalyst, and product, can be misleading, and its industrial application is likely to produce devastation and pollution along with the intended product. In nature and industry, the reactants, products, and energy changes are linked to the ecology and to the economy. In physiological chemistry, events in the organism are linked to the environment so closely that food, water, air, soil, and pollution form a firmly linked functional system.
 But “medical physiology” has evolved as a separate thing, in which formulas that describe specific situations are linked to each other by fragmentary schemes, terminology, and computer models. This jerrybuilt scheme is even more roughly set into a hypothetical environment of “the origin of life,” “evolution,” “inheritance,” “society,” and a few other perfunctory contextualizations that have no more relevance to the subject than do the literary epigraphs that are often used at the beginning of chapters in medical books, to signify that the author isn’t just a technical hack.
 This physiological mythology has made possible a practice of medicine in which “genes” and “a virus” are regularly invoked to explain things that can’t be remedied, and in which any fleshy body is described as “well nourished,” and in which malnutrition and poisoning by pollutants are systematically dismissed as explanations for sicknesses, while thousands of different drugs are administered according to instructions given by their salesmen. It is also deeply linked to attitudes that have turned the practice of medicine into the surest way for an individual to get rich and retire early. It creates a sense of confidence that the physician is doing the right thing, because there is a little physiological rationale for everything. When a practice is replaced by its opposite, there is also a rationale for that. In fact, medical textbooks are written to rationalize the highly arbitrary practices of the industry. If, for some reason, perpetual motion machines had been as successful economically as steam engines were, laws of thermodynamics would have been written to describe them, just as thermodynamic laws were invented to describe the theory of steam engines.
```

Comment:
```suggestion
This ---- physiological mythology has made possible a practice of medicine in which “genes” and “a virus” are regularly invoked to explain things that can’t be remedied, and in which any fleshy body is described as “well nourished,” and in which malnutrition and poisoning by pollutants are systematically dismissed as explanations for sicknesses, while thousands of different drugs are administered according to instructions given by their salesmen. It is also deeply linked to attitudes that have turned the practice of medicine into the surest way for an individual to get rich and retire early. It creates a sense of confidence that the physician is doing the right thing, because there is a little physiological rationale for everything. When a practice is replaced by its opposite, there is also a rationale for that. In fact, medical textbooks are written to rationalize the highly arbitrary practices of the industry. If, for some reason, perpetual motion machines had been as successful economically as steam engines were, laws of thermodynamics would have been written to describe them, just as thermodynamic laws were invented to describe the theory of steam engines.
```